src/arcam/fmj/__init__.py

```diff
@@ -387,6 +394,26 @@ class DecodeModeMCH(IntOrTypeEnum):
         SourceCodes.SAT: bytes([16, 20]),
         SourceCodes.VCR: bytes([16, 21]),
         SourceCodes.FOLLOW_ZONE_1: bytes([16, 20])
+    },
+    (ApiModel.APISA_SERIES, 1): {
```

Comment:
The values of these can be verified on the [SA series PDF](https://www.arcam.co.uk/ugc/tor/SA10/Custom%20Installation%20Notes/SH277E_RS232_SA10_SA20_B.pdf) page 19 and [SA30 PDF](http://www.arcam.co.uk/ugc/tor/SA30/Custom%20Installation%20Notes/SH306E_RS232_SA30_4.pdf) page 26